src/plugins/intel_gpu/src/graph/common_utils/jitter.hpp

```diff
@@ -220,12 +221,27 @@ class JitTerm {
     std::string text;
 };
 
+inline bool is_number(const JitTerm& s) {
+    return !s.str().empty() && std::all_of(s.str().begin(), s.str().end(), ::isdigit);
```

Comment:
what about sign str? (e.g., -)

Reply:
It's mainly used for shapes for now which are positive numbers, but we can extend it in the future for negatives and fp values if needed